content/en/docs/concepts/configuration/secret.md

```diff
@@ -339,9 +339,15 @@ files.
 
 When a secret being already consumed in a volume is updated, projected keys are eventually updated as well.
 Kubelet is checking whether the mounted secret is fresh on every periodic sync.
-However, it is using its local ttl-based cache for getting the current value of the secret.
-As a result, the total delay from the moment when the secret is updated to the moment when new keys are
-projected to the pod can be as long as kubelet sync period + ttl of secrets cache in kubelet.
+However, it is using its local cache for getting the current value of the secret.
+The type of the cache is configurable (`ConfigMapAndSecretChangeDetectionStrategy` field in
+[KubeletConfiguration struct](https://github.com/kubernetes/kubernetes/blob/release-1.12/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go))
```

Comment:
I have the same concern here as in https://github.com/kubernetes/website/pull/10087/files#r212857293
/cc @chenopis

Reply:
Good catch, @xiangpengzhao! :sparkles:

@wojtek-t Please take a look at the linked PR :point_up: and update the file path here. Thanks in advance!

Reply:
Done